src/key.rs

```diff
@@ -372,14 +346,6 @@ impl SecretKey {
     }
 }
 
-#[cfg(feature = "hashes")]
-impl<T: ThirtyTwoByteHash> From<T> for SecretKey {
```

Comment:
In 00b66df56d0383ae4090b98cdd515660a98ab1f1:

We should wait til after the deprecation cycle to drop these From impls. If we drop them now that has the same effect as just deleting the crate -- sudden compilation failures without a clear path forward.